.github/workflows/release.yml

```
@@ -2,12 +2,14 @@ name: "Release"
 on:
   release:
     types: [published]
-env:
-  GIT_USER_NAME: ${{ vars.GIT_USER_NAME }}
-  GIT_USER_EMAIL: ${{ vars.GIT_USER_EMAIL }}
 jobs:
   release:
+    name: "Publish Release"
     runs-on: ubuntu-24.04
+    permissions:
+      packages: read  # for pre-release workflow
+      contents: write  # to create release
+      issues: write  # to modify milestones
```

I can't recall if I kept the milestone support in the files, if I did, I likely made it so that it silently fails if it doesn't exist or has a problem.  We should probably test this.